### PR TITLE
Added find to Nano

### PR DESCRIPTION
**Pull request by Panickev**
## Overview

Added find to Nano. This way you can use custom queries.

### CODE REVIEW

File: lib/nano.js [hunk 1]
```diff
@@ -458,6 +458,10 @@ module.exports = exports = nano = function dbScope(cfg) {
 
       return relax({db: dbName, doc: docName, qs: qs}, callback);
     }
+	
+	function findDoc(query, callback) {
+		return relax({ db: dbName, path: '_find', method: 'POST', body: query }, callback);
```
Comment (garrensmith) [inline lib/nano.js]: Can you encode the url.

File: lib/nano.js [hunk 2]
```diff
@@ -773,6 +777,7 @@ module.exports = exports = nano = function dbScope(cfg) {
       session: session,
       insert: insertDoc,
       get: getDoc,
+	  find: findDoc,
```
Comment (garrensmith) [inline lib/nano.js]: Could you align this correctly.

File: lib/nano.js (hunk 1 above)
Comment (janl) [inline lib/nano.js]: none of the other methods here do this at this point, I assume `relax()` does this already.

**Comment by janl** [comment]
Great start, closing in favour of #50 which adds tests.

**Pull request closed without merging**